be/src/exec/parquet_writer.h

```diff
@@ -42,6 +42,7 @@ struct TableInfo {
     bool _enable_dictionary = true;
     std::string _partition_location = "";
     std::shared_ptr<::parquet::schema::GroupNode> _schema;
+    TCloudConfiguration _cloud_conf;
```

Comment:
The filed name in struct don't need the leading `_`. You can name it clould_conf

Reply:
removed leading underscore of all fields